grafana/README.md

```diff
@@ -0,0 +1,22 @@
+# Grafana
+
+This directory contains useful out-of-box dashboards.
+
+# Constraints
+
+Dashboards contain [summary](../grafana/dashboards/summary) and [single](../grafana/dashboards/single) dashboards.
+* summary: Aggregated dashboards for a cluster.
+* single: dashboards for a single resource(e.g. `node` or `pod`).
+
+**Notice**
+* summary dashboards use `alias` as the `cluster` template variable.
```

Comment:
@brancz I totally agree with you. However, I do not know how to implement `if single cluster, do not show up cluster template var; otherwise, do it` in Grafana. 

If there is no such an easy way to implement this, how about we document `cluster` template var clearly and make it all the time.